.platform.app.yaml

```
@@ -43,6 +47,7 @@ variables:
         # Change this if you use a different env then "prod", if you change this to "dev" remove "--no-dev" in build step
```

```suggestion
        # Change this if you use a different env then "prod", if you change this to "dev" remove "--no-dev"
        # and comment out 'rm web/app_dev.php' in build step
```

---

@reithor Actually you don't need to comment out `rm web/app_dev.php`, we handle all dev stuff in `app.php`, while `app_dev.php` is just convince to now have to set ENV, which you anyway will have to do here.

```suggestion
       # Change this if you use a different env then "prod", if you change this to "dev" remove "--no-dev" from `composer install` command.
```

ok? _(assumes we go back to build flavour:none and reinstate own call to composer install, otherwise this section should be removed actually)_